src/Viewer/components/Monaco/MonacoInstance.js

```diff
@@ -272,23 +257,22 @@ function MonacoInstance ({
         };
     }, []);
 
-    const monacoOptions = {
-        "readOnly": true,
-        "renderWhitespace": "none",
-        "wordWrap": "on",
-        "scrollBeyondLastLine": false,
-    };
+    useEffect(() => {
+        if (null === editorRef.current) {
+            handleEditorWillMount();
+            editorRef.current = monaco.editor.create(editorContainerRef.current, {
+                language: "logLanguage",
+                readOnly: true,
+                renderWhitespace: "none",
+                wordWrap: "on",
+                scrollBeyondLastLine: false,
+                theme: (theme === THEME_STATES.LIGHT) ? "customLogLanguageLight" : "customLogLanguageDark",
+            });
+            handleEditorDidMount();
+        }
+    }, [])
 
-    return (
-        <Editor
-            defaultValue="Loading content..."
-            theme={monacoTheme}
-            language={language}
-            beforeMount={handleEditorWillMount}
-            onMount={handleEditorDidMount}
-            options={monacoOptions}
-        />
-    );
+    return (<div ref={editorContainerRef} style={{ display: 'flex', width: '100% ', height: "100%" }}></div>)
```

Comment:
Question: Do we have to specify the width and height to be 100% of the parent container size? Just curious if the monaco.editor.create() function will set the styles on the DOM.

Reply:
Without the manually configure the style, the editor is not shown properly :(
See the attachments after I commented out the width and height
![Screenshot_20240119_171009](https://github.com/y-scope/yscope-log-viewer/assets/50559854/c942863b-1551-4b07-b5eb-f125e96e3a8a)

Reply:
Gotcha. Let's keep the styles then. Please consider putting the styles for a class into a .css file and set the classname on the component.

Reply:
Should we create a new .css file or use the MonacoInstance.scss?

Reply:
MonacoInstance.scss please.
We can name it like `monaco-container`